### ISSUE TITLE
[API] Warning

### ISSUE THREAD

**Issue by wukongrita** [simplabs/ember-simple-auth#1615]
Hi, just installed this ESA and got the following warning:

```
$ ember s
[API] Warning: The .read and .rebuild APIs will stop working in the next Broccoli version
[API] Warning: Use broccoli-plugin instead: https://github.com/broccolijs/broccoli-plugin
[API] Warning: Plugin uses .read/.rebuild API: Filter
```

Any way how should I fix them?

**Comment by wukongrita** [comment]
@marcoow forgot to info:

```
$ ember -v
ember-cli: 3.1.4
node: 8.9.4
os: darwin x64
```

**Comment by bartsqueezy** [comment]
@wukongrita, see #1601.  Once that is merged and a new version is released, that message should disappear.

**Comment by fenichelar** [comment]
This looks fixed in master, just waiting on a new release.

**Comment by marcoow** [comment]
closed via #1601

**Comment by marcoow** [comment]
closed via #1601

**Comment by fenichelar** [comment]
Any update on when a release will be made that includes this?
